src/main.cpp

```diff
@@ -1,13 +1,25 @@
 #include <QGuiApplication>
 #include <QQmlApplicationEngine>
+#include "databaseengine.h"
+
+static QObject *BD_Create(QQmlEngine *, QJSEngine *)
```

Comment:
в нашем стиле мы не называем методы с большой буквы
```suggestion
static QObject *dataBaseFactory(QQmlEngine *, QJSEngine *)
```